pom.xml

```diff
@@ -433,7 +433,7 @@
     <!-- Version of the KIE Workbench application. Shown for example in About dialog. Will be usually overridden by productisation. -->
     <version.org.kie.workbench.app>${version.org.kie}</version.org.kie.workbench.app>
 
-    <version.thorntail>2.7.0.Final</version.thorntail>
+    <version.bootable>4.0.3.Final</version.bootable>
```

Comment:
@mareknovotny I'm setting the version here. Removed the rest during clean up.

Reply:
```suggestion
    <version.bootable>4.1.0.Final</version.bootable>
```